src/store/proposals/sagas/proposalList.ts

```diff
@@ -22,7 +22,7 @@ import { proposalsActions } from '../proposals.slice';
 import { governorContractsSelector } from '../proposals.selectors';
 import { MulticallProviderType } from '../../types';
 
-function* fetchProposalStates(
+export function* fetchProposalStates(
```

Comment:
Maybe let's create an `utils` file in this directory and put all reusable sagas there